src/Microsoft.Restier.WebApi/HttpConfigurationExtensions.cs

```diff
@@ -65,6 +72,11 @@ public static class HttpConfigurationExtensions
 
                 var route = config.MapODataServiceRoute(
                     routeName, routePrefix, model, new DefaultODataPathHandler(), conventions, batchHandler);
+
+                // Customized converter should be added in ConfigureApi as service
+                var converter = api.Context.GetApiService<ODataPayloadValueConverter>();
```

Comment:
Yeah, this also help to resolve an issue of some our test cases which calls Api configuration before calling map route, and cause null reference exception (no exception in normal case which calls map route first)